src/components/browser-banner/_macro.spec.js

```diff
@@ -5,102 +5,101 @@ import * as cheerio from 'cheerio';
 import axe from '../../tests/helpers/axe';
 import { renderComponent } from '../../tests/helpers/rendering';
 
-const EXAMPLE_BROWSER_BANNER_DEFAULT = {};
-
-describe('macro: browser-banner', () => {
-    it('passes jest-axe checks with', async () => {
-        const $ = cheerio.load(renderComponent('browser-banner', EXAMPLE_BROWSER_BANNER_DEFAULT));
-
-        const results = await axe($.html());
-        expect(results).toHaveNoViolations();
-    });
-
-    it('has the default `bannerLeadingText`', () => {
-        const $ = cheerio.load(renderComponent('browser-banner', EXAMPLE_BROWSER_BANNER_DEFAULT));
-
-        const bannerLeadingText = $('.ons-browser-banner__lead').text().trim();
-        expect(bannerLeadingText).toBe('This website no longer supports your browser.');
-    });
-
-    it('has the default `bannerCTA`', () => {
-        const $ = cheerio.load(renderComponent('browser-banner', EXAMPLE_BROWSER_BANNER_DEFAULT));
-
-        const bannerCtaHtml = $('.ons-browser-banner__cta').text().trim();
-        expect(bannerCtaHtml).toBe('You can upgrade your browser to the latest version.');
-    });
-
-    it('has the default `bannerLinkUrl`', () => {
-        const $ = cheerio.load(renderComponent('browser-banner', EXAMPLE_BROWSER_BANNER_DEFAULT));
-
-        expect($('.ons-browser-banner__link').attr('href')).toBe('https://www.ons.gov.uk/help/browsers');
-    });
-
-    it('has `container--wide` class when `wide` is true', () => {
-        const $ = cheerio.load(
-            renderComponent('browser-banner', {
-                ...EXAMPLE_BROWSER_BANNER_DEFAULT,
-                wide: true,
-            }),
-        );
-
-        expect($('.ons-container').hasClass('ons-container--wide')).toBe(true);
+describe('FOR: browser-banner', () => {
+    describe('GIVEN: Params: default', () => {
+        describe('WHEN: params are at default and language is default', () => {
```

Comment:
```suggestion
        describe('WHEN: params are at default state', () => {
```
Language is a param so its covered by the first part of the statement